.github/workflows/ci.yml

```diff
@@ -4,7 +4,6 @@ name: Angular CI
 
 on:
   push:
-    branches: [ main, develop ]
```

Comment:
pls explain to me

Reply:
I removed the reference to specific branches to make this workflow run on every push, no matter on which branch.